scripts/sct_register_to_template.py

```diff
@@ -72,133 +75,210 @@ def __init__(self):
+            "To register a subject to the template, try the default command:\n"
+            "  sct_register_to_template -i data.nii.gz -s data_seg.nii.gz -l data_labels.nii.gz\n"
+            "\n"
+            "If this default command does not produce satisfactory results, please refer to:\n"
+            "  https://sourceforge.net/p/spinalcordtoolbox/wiki/registration_tricks/\n"
+            "\n"
+            "The default registration method brings the subject image to the template, which can be problematic with "
+            "highly non-isotropic images as it would induce large interpolation errors during the straightening "
+            "procedure. Although the default method is recommended, you may want to register the template to the "
+            "subject (instead of the subject to the template) by skipping the straightening procedure. To do so, use "
+            "the parameter '-ref subject'. Example below:\n"
+            "  sct_register_to_template -i data.nii.gz -s data_seg.nii.gz -l data_labels.nii.gz -ref subject -param "
+            "step=1,type=seg,algo=centermassrot,smooth=0:step=2,type=seg,algo=columnwise,smooth=0,smoothWarpXY=2\n"
+            "\n"
+            "Vertebral alignment (step 2) consists in aligning the vertebrae between the subject and the template. "
+            "Two types of labels are possible:\n"
+            "  - Vertebrae mid-body labels, created at the center of the spinal cord using the parameter '-l';\n"
+            "  - Posterior edge of the intervertebral discs, using the parameter '-ldisc'.\n"
+            "\n"
+            "If only one label is provided, a simple translation will be applied between the subject label and the "
+            "template label. No scaling will be performed. \n"
+            "\n"
+            "If two labels are provided, a linear transformation (translation + rotation + superior-inferior linear "
+            "scaling) will be applied. The strategy here is to defined labels that cover the region of interest. For "
+            "example, if you are interested in studying C2 to C6 levels, then provide one label at C2 and another at "
+            "C6. However, note that if the two labels are very far apart (e.g. C2 and T12), there might be a "
+            "mis-alignment of discs because a subject''s intervertebral discs distance might differ from that of the "
+            "template.\n"
+            "\n"
+            "If more than two labels (only with the parameter '-disc') are used, a non-linear registration will be "
+            "applied to align the each intervertebral disc between the subject and the template, as described in "
+            "sct_straighten_spinalcord. This the most accurate and preferred method. This feature does not work with "
+            "the parameter '-ref subject'.\n"
+            "\n"
+            "More information about label creation can be found at "
+            "https://www.slideshare.net/neuropoly/sct-course-20190121/42"
```

Comment:
The link is dead. I know this alternative is not great (icloud not accessible everywhere), but until we have a proper documentation on spinalcordmri.com, that's probably the best we can do (?)
```suggestion
            "https://www.icloud.com/keynote/0th8lcatyVPkM_W14zpjynr5g#SCT%5FCourse%5F20200121 (p47)"
```

Reply:
Ahh, it looks like this is related to #2134. Good catch.